pkg/operator/ceph/csi/util.go

```diff
@@ -84,6 +84,20 @@ func templateToDeployment(name, templateData string, p templateParam) (*apps.Dep
 	return &dep, nil
 }
 
+func templateTologrotateSidecar(name, templateData string, p templateParam) (corev1.Container, error) {
+	var container corev1.Container
+	t, err := loadTemplate(name, templateData, p)
+	if err != nil {
+		return container, errors.Wrap(err, "failed to load logrotate container template")
```

Comment:
Just panic. Since the template is internal and no user input, we can just fail completely. This simplifies the method because no need to return an error.
```suggestion
		panic(errors.Wrap(err, "failed to load logrotate container template"))
```